lib/superintendent/request/validator.rb

```diff
@@ -14,15 +14,17 @@ class Validator
     }.freeze
 
     DEFAULT_OPTIONS = {
-      :monitored_content_types => ['application/json']
+      :monitored_content_types => ['application/json'],
+      :error_class => Superintendent::Request::Error
```

Comment:
Hmm, hash style has definitely been inconsistent up to this point. Let's make it consistent with the "new" style.

Reply:
Just to be consistent with the existing code. 👍 will do.